src/components/deploymentConfig/DeploymentTemplateView/DeploymentTemplateView.component.tsx

```diff
@@ -424,3 +577,30 @@ export const DeleteOverrideDialog = ({ appId, envId, initialise }) => {
         </ConfirmationDialog>
     )
 }
+
+export function DropdownContainer({ isOpen, onClose, children }) {
+    if (!isOpen) {
+        return null
+    }
+
+    return (
+        <div className='dc__transparent-div' onClick={onClose}>
```

Comment:
Please use " instead of ' in html attributes

Reply:
done